### PR TITLE
Reviewed distributor index and configuration

### PR DESCRIPTION
**Pull request by danielmarbach**
Connects to https://github.com/Particular/PlatformDevelopment/issues/1355

### CODE REVIEW

File: transports/msmq/distributor/error-handling.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Immediate Retries are always performed on the worker.
 
 The behavior of Delayed Retries is different between major versions.
 
-Delayed Retries has two configurable items TimeIncrease and NumberOfRetries.
+Delayed Retries has two configurable items *TimeIncrease* and *NumberOfRetries*.
```
Comment (danielmarbach) [inline transports/msmq/distributor/error-handling.md]: @SzymonPobiega can you help me understand the note in this file:

> NOTE: If settings are managed in XML configuration a comment stating that changes should be updated on the distributor and worker configuration files.

I wasn't able to grasb this

Comment (SzymonPobiega) [inline transports/msmq/distributor/error-handling.md]: @danielmarbach I have no idea what does it mean :-(

File: transports/msmq/distributor/configuration.md [hunk 2]
```diff
@@ -74,4 +75,4 @@ If self-hosting the endpoint here is the code required to enlist the endpoint wi
 
 snippet: ConfiguringWorker
 
-Similar to self hosting, ensure the `app.config` of the worker contains the `MasterNodeConfig` section to point to the host name where the distributor process is running.
+Similar to self-hosting, ensure the `app.config` of the worker contains the `MasterNodeConfig` section to point to the hostname where the distributor process is running.
```
Comment (SimonCropp) [inline transports/msmq/distributor/configuration.md]: > Similar to self hosting, ensure the `app.config` of the worker contains the `MasterNodeConfig` section

I think configuring that in app.config  is redundant?  https://docs.particular.net/transports/msmq/distributor/configuration#versions-nservicebus-6-0-and-above

File: transports/msmq/distributor/error-handling.md (hunk 1 above)
Comment (SimonCropp) [inline transports/msmq/distributor/error-handling.md]: i think it means something like  "any change in distributor configuration needs to be replicated to all nodes"?

File: transports/msmq/distributor/index.md [hunk 3]
```diff
@@ -15,29 +16,28 @@ redirects:
 
 The NServiceBus Distributor is similar in behavior to standard [load balancers](https://en.wikipedia.org/wiki/Load_balancing_%28computing%29). It is the key to transparently scaling out message processing over many machines.
 
-As a standard NServiceBus process, the Distributor maintains all the fault-tolerant and performance characteristics of NServiceBus but is designed never to overwhelm any of the worker nodes configured to receive work from it.
+As a standard NServiceBus process, the Distributor maintains all the fault-tolerant and performance characteristics of NServiceBus but is designed to never overwhelm any of the worker nodes configured.
```
Comment (SimonCropp) [inline transports/msmq/distributor/index.md]: "configured" is redundant?

**Pull request merged**